<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AppointmentDialog</class>
 <widget class="QDialog" name="AppointmentDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>595</width>
    <height>509</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>New Appointment - Daylight</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>2</number>
   </property>
   <property name="leftMargin">
    <number>4</number>
   </property>
   <property name="topMargin">
    <number>2</number>
   </property>
   <property name="rightMargin">
    <number>4</number>
   </property>
   <property name="bottomMargin">
    <number>2</number>
   </property>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="tab_basic">
      <attribute name="title">
       <string>Basic</string>
      </attribute>
      <layout class="QFormLayout" name="formLayout_2">
       <item row="0" column="0">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Title</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="edReason"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Calendar</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QComboBox" name="comboSelectCalendar"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>From</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QDateTimeEdit" name="dateTimeStartInterval">
         <property name="correctionMode">
          <enum>QAbstractSpinBox::CorrectToNearestValue</enum>
         </property>
         <property name="displayFormat">
          <string>dd.MM.yyyy HH:mm</string>
         </property>
         <property name="calendarPopup">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>To</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QDateTimeEdit" name="dateTimeEndInterval">
         <property name="correctionMode">
          <enum>QAbstractSpinBox::CorrectToNearestValue</enum>
         </property>
         <property name="displayFormat">
          <string>dd.MM.yyyy HH:mm</string>
         </property>
         <property name="calendarPopup">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Repeat...</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QComboBox" name="comboRepeat">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>4</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Reminder</string>
         </property>
        </widget>
       </item>
       <item row="7" column="1">
        <widget class="QComboBox" name="comboReminder">
         <property name="enabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="8" column="1">
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="6" column="1">
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_recurrence">
      <attribute name="title">
       <string>Recurrence</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="leftMargin">
        <number>4</number>
       </property>
       <property name="topMargin">
        <number>4</number>
       </property>
       <property name="rightMargin">
        <number>4</number>
       </property>
       <property name="bottomMargin">
        <number>4</number>
       </property>
       <item>
        <widget class="QToolBox" name="toolBox">
         <property name="currentIndex">
          <number>5</number>
         </property>
         <property name="tabSpacing">
          <number>2</number>
         </property>
         <widget class="QWidget" name="rec_page_bym">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>483</width>
            <height>248</height>
           </rect>
          </property>
          <attribute name="label">
           <string>By Month</string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout">
           <item row="0" column="0">
            <widget class="QCheckBox" name="cb_bm_jan">
             <property name="text">
              <string>Jan</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QCheckBox" name="cb_bm_feb">
             <property name="text">
              <string>Feb</string>
             </property>
            </widget>
           </item>
           <item row="0" column="2">
            <widget class="QCheckBox" name="cb_bm_mar">
             <property name="text">
              <string>Mar</string>
             </property>
            </widget>
           </item>
           <item row="0" column="3">
            <widget class="QCheckBox" name="cb_bm_apr">
             <property name="text">
              <string>Apr</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QCheckBox" name="cb_bm_may">
             <property name="text">
              <string>May</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QCheckBox" name="cb_bm_jun">
             <property name="text">
              <string>Jun</string>
             </property>
            </widget>
           </item>
           <item row="2" column="2">
            <widget class="QCheckBox" name="cb_bm_jul">
             <property name="text">
              <string>Jul</string>
             </property>
            </widget>
           </item>
           <item row="2" column="3">
            <widget class="QCheckBox" name="cb_bm_aug">
             <property name="text">
              <string>Aug</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QCheckBox" name="cb_bm_sep">
             <property name="text">
              <string>Sep</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QCheckBox" name="cb_bm_oct">
             <property name="text">
              <string>Oct</string>
             </property>
            </widget>
           </item>
           <item row="3" column="2">
            <widget class="QCheckBox" name="cb_bm_nov">
             <property name="text">
              <string>Nov</string>
             </property>
            </widget>
           </item>
           <item row="3" column="3">
            <widget class="QCheckBox" name="cb_bm_dec">
             <property name="text">
              <string>Dec</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="rec_page_bwn">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>483</width>
            <height>248</height>
           </rect>
          </property>
          <attribute name="label">
           <string>By Week No</string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout_4">
           <item row="0" column="0">
            <widget class="QSpinBox" name="sb_bw_weekno">
             <property name="prefix">
              <string>Week </string>
             </property>
             <property name="minimum">
              <number>-53</number>
             </property>
             <property name="maximum">
              <number>53</number>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QPushButton" name="pb_bw_add">
             <property name="text">
              <string>Add</string>
             </property>
            </widget>
           </item>
           <item row="0" column="2">
            <widget class="QPushButton" name="pb_bw_remove">
             <property name="text">
              <string>Remove</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0" colspan="3">
            <widget class="QListWidget" name="lw_bw_list"/>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="rec_page_byd">
          <attribute name="label">
           <string>By Year Day</string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout_3">
           <item row="0" column="0">
            <widget class="QSpinBox" name="sb_byd_yearday">
             <property name="suffix">
              <string/>
             </property>
             <property name="prefix">
              <string>Day </string>
             </property>
             <property name="minimum">
              <number>-366</number>
             </property>
             <property name="maximum">
              <number>366</number>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QPushButton" name="bp_byd_add">
             <property name="text">
              <string>Add</string>
             </property>
            </widget>
           </item>
           <item row="0" column="2">
            <widget class="QPushButton" name="bp_byd_remove">
             <property name="text">
              <string>Remove</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0" colspan="3">
            <widget class="QListWidget" name="lw_byd_list"/>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="rec_page_bymday">
          <attribute name="label">
           <string>By Monthday </string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="0" column="0">
            <widget class="QSpinBox" name="sb_bymday_monthday">
             <property name="prefix">
              <string>Day </string>
             </property>
             <property name="minimum">
              <number>-31</number>
             </property>
             <property name="maximum">
              <number>31</number>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QPushButton" name="pb_bymday_add">
             <property name="text">
              <string>PushButton</string>
             </property>
            </widget>
           </item>
           <item row="0" column="2">
            <widget class="QPushButton" name="bp_bymday_remove">
             <property name="text">
              <string>PushButton</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0" colspan="3">
            <widget class="QListWidget" name="lw_bymday_list"/>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="rec_page_bd">
          <attribute name="label">
           <string>By Day</string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout_5">
           <item row="0" column="0">
            <widget class="QSpinBox" name="sb_byday_daynum">
             <property name="suffix">
              <string>.</string>
             </property>
             <property name="prefix">
              <string>The </string>
             </property>
             <property name="minimum">
              <number>-53</number>
             </property>
             <property name="maximum">
              <number>53</number>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QComboBox" name="cb_byday_dayname">
             <item>
              <property name="text">
               <string>Monday</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Tuesday</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Wednesday</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Thursday</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Friday</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Satueday</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Sunday</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="0" column="2">
            <widget class="QPushButton" name="pb_byday_add">
             <property name="text">
              <string>Add</string>
             </property>
            </widget>
           </item>
           <item row="0" column="3">
            <widget class="QPushButton" name="pb_byday_remove">
             <property name="text">
              <string>PushButton</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0" colspan="4">
            <widget class="QListWidget" name="lw_byday_list"/>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="rec_page_misc">
          <attribute name="label">
           <string>Misc</string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout_7">
           <item row="1" column="2">
            <widget class="QDateTimeEdit" name="dateTimeRepeatUntil">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="correctionMode">
              <enum>QAbstractSpinBox::CorrectToNearestValue</enum>
             </property>
             <property name="showGroupSeparator" stdset="0">
              <bool>false</bool>
             </property>
             <property name="displayFormat">
              <string>dd.MM.yyyy HH:mm</string>
             </property>
             <property name="calendarPopup">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="label_8">
             <property name="text">
              <string>Week starts</string>
             </property>
            </widget>
           </item>
           <item row="0" column="2">
            <widget class="QComboBox" name="cb_byday_dayname_2">
             <item>
              <property name="text">
               <string>Monday</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Tuesday</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Wednesday</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Thursday</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Friday</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Satueday</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Sunday</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_9">
             <property name="text">
              <string>Repeat</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0" colspan="3">
            <widget class="QGroupBox" name="groupBox">
             <property name="title">
              <string>Setpos-Day</string>
             </property>
             <layout class="QGridLayout" name="gridLayout_6">
              <property name="leftMargin">
               <number>2</number>
              </property>
              <property name="topMargin">
               <number>2</number>
              </property>
              <property name="rightMargin">
               <number>2</number>
              </property>
              <property name="bottomMargin">
               <number>2</number>
              </property>
              <property name="spacing">
               <number>2</number>
              </property>
              <item row="0" column="0">
               <widget class="QSpinBox" name="sb_setposday_days">
                <property name="minimum">
                 <number>-366</number>
                </property>
                <property name="maximum">
                 <number>366</number>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QPushButton" name="pb_setposday_add">
                <property name="text">
                 <string>PushButton</string>
                </property>
               </widget>
              </item>
              <item row="0" column="2">
               <widget class="QPushButton" name="pb_setposday_remove">
                <property name="text">
                 <string>PushButton</string>
                </property>
               </widget>
              </item>
              <item row="1" column="0" colspan="3">
               <widget class="QListWidget" name="lw_setposday_list"/>
              </item>
             </layout>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QCheckBox" name="checkForever">
             <property name="text">
              <string>forever</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_alarm">
      <attribute name="title">
       <string>Alarm</string>
      </attribute>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>dateTimeEndInterval</tabstop>
  <tabstop>edReason</tabstop>
  <tabstop>comboSelectCalendar</tabstop>
  <tabstop>comboRepeat</tabstop>
  <tabstop>dateTimeStartInterval</tabstop>
  <tabstop>comboReminder</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>AppointmentDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>257</x>
     <y>294</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>AppointmentDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>325</x>
     <y>294</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
